labs/bbb-webrtc-sfu/lib/bbb/messages/Constants.js

```diff
@@ -58,7 +62,6 @@ const config = require('config');
         STOP_TRANSCODER_REQ_2x: "StopTranscoderSysReqMsg",
         STOP_TRANSCODER_RESP_2x: "StopTranscoderSysRespMsg",
 
-        USER_CAM_BROADCAST_STARTED_2x: "UserBroadcastCamStartMsg",
         USER_CAM_BROADCAST_STOPPED_2x: "UserBroadcastCamStopMsg",
```

Comment:
Looks like `USER_CAM_BROADCAST_STOPPED_2x: "UserBroadcastCamStopMsg",` could be removed too, no?